index.js

```diff
@@ -1,18 +1,12 @@
 module.exports = function (hljs) {
   const KEYWORDS =
-    "as assert case const external fn if import let " +
-    "panic opaque pub todo tuple type use";
+    "as assert case const fn if import let panic use opaque pub todo type";
   const STRING = {
     className: "string",
     variants: [{ begin: /"/, end: /"/ }],
     contains: [hljs.BACKSLASH_ESCAPE],
     relevance: 0,
   };
-  const NAME = {
```

Comment:
What was this for? Should it be gone or does it miss in the website version of this file?

Reply:
okay, I will use the website version but not remove the NAME part here and update the website version with the name part.